airbyte-integrations/connectors/source-sendgrid/unit_tests/unit_test.py

```diff
@@ -19,6 +23,13 @@ def sendgrid_stream_fixture(mocker) -> SendgridStream:
     return SendgridStream()  # type: ignore
 
 
+@pytest.fixture()
+def mock_pendulum_now(monkeypatch):
+    pendulum_mock = unittest.mock.MagicMock(wraps=pendulum.now)
+    pendulum_mock.return_value = FAKE_NOW
+    monkeypatch.setattr(pendulum, "now", pendulum_mock)
```

Comment:
I like to use FreezeGun for this kind of mock :D https://github.com/spulec/freezegun